src/cli/serve/serve.js

```diff
@@ -1,6 +1,8 @@
 import _ from 'lodash';
-const { isWorker } = require('cluster');
-const { resolve } = require('path');
+import { isWorker } from 'cluster';
+import { resolve } from 'path';
+import Config from '../../server/config/config';
+import loggingConfiguration from '../../server/logging/configuration';
```

Comment:
Mind if the server just export a method for this? I'd prefer that the cli module not reach into the server module like this.

```
server.applyNewLoggingSettings(getCurrentSettings().logging);
```